### PR TITLE
Cleaner Docker file with much lighter base image

### PR DESCRIPTION
**Pull request by hosseinabaiyani**
I've rewritten the Dockerfile with cleaner structure and lighter base image

### CODE REVIEW

**Comment by hosseinabaiyani** [comment]
i just wanted to know why you closed this pull request

**Comment by itspooya** [comment]
Looks good to me
@MHSanaei

**Comment by MHSanaei** [comment]
thank you, but when I saw that iran.dat was deleted, I decided to close the PR

**Comment by hosseinabaiyani** [comment]
@MHSanaei I've added iran.dat
please reopen and merge it ;)

File: DockerInit.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    FNAME="arm64";
+else
+    ARCH="64";
+    FNAME="amd64";
+fi
+mkdir -p build/bin
+cd build/bin
+wget "https://github.com/mhsanaei/xray-core/releases/latest/download/Xray-linux-${ARCH}.zip"
+unzip "Xray-linux-${ARCH}.zip"
+rm -f "Xray-linux-${ARCH}.zip" geoip.dat geosite.dat
```
Comment (MHSanaei) [inline DockerInit.sh]: you forgot iran.dat here

Comment (hosseinabaiyani) [reply DockerInit.sh]: Fixed it

**Pull request merged**